### DISCUSSION TITLE
HOW DO I IMPORT PNG TEXTURES USING UABE?

### DISCUSSION THREAD

**Discussion by DraKoNiier** [nesrak1/UABEA#362] [Q&A]
Hello, I want to edit a mod for a certain game that, for reasons of not annoying the big N, I'm going to call P*k*****.

I open the file where the texture is, go to info, go to the texture I want to replace and when I click on plugins, only a blank window appears, with no option to import a texture in png. 

The goal is to replace a button sprite with another one. 
With texture2D type file, I can replace it easily.

Print:
![Captura de tela 2024-07-28 004418](https://github.com/user-attachments/assets/68ad323f-3223-40cc-893d-7fde44ef7dde)

NOTE: The file type is a sprite.

Thank you in advance and if there is another app that can do this for me, I appreciate the recommendation.

**Comment by nesrak1** [comment]
For various reasons, sprite import is complicated and not supported as of now. You can always edit the texture2d that belongs to this sprite (which you can find in the view data window under m_RD.texture, then right click the texture base field and click "visit asset")
